pkg/controller/cassandradatacenter/configmap.go

```diff
@@ -60,14 +58,50 @@ func createOrUpdateOperatorConfigMap(rctx *reconciliationRequestContext, seedNod
 	return configVolume, nil
 }
 
-func addCassandraGossipingPropertyFileSnitchProperties(cdc *cassandraoperatorv1alpha1.CassandraDataCenter, addFileFn func(path string, data string)) {
+func createOrUpdateCassandraRackConfig(rctx *reconciliationRequestContext, rack string) (*corev1.Volume, error) {
```

Comment:
Should the `rack` parameter be of type `Rack`?

Reply:
switched to *Rack for all function calls.